src/v4/component/CommunityStories/index.tsx

```diff
@@ -35,12 +33,21 @@ const CommunityStories = ({
   const navigation =
     useNavigation() as NativeStackNavigationProp<RootStackParamList>;
   const styles = useStyles();
-  const { client } = useAuth();
+  const { getUiKitConfig } = useConfig();
   const hasStoryPermission = useStoryPermission(communityId);
-  const userId = (client as Amity.Client).userId;
-  const { getStories, stories, loading } = useStory();
+  const { loading, getStoryTarget, storyTarget } = useStory();
   const { getImage } = useFile();
   const [avatarUrl, setAvatarUrl] = useState(undefined);
+  const [viewStory, setViewStory] = useState(false);
+  const storyRingColor: string[] = storyTarget?.hasUnseen
+    ? (getUiKitConfig({
+        page: PageID.StoryPage,
+        component: ComponentID.StoryTab,
+        element: ElementID.StoryRing,
+      })?.progress_color as string[]) ?? ['#e2e2e2', '#e2e2e2']
+    : storyTarget?.failedStoriesCount > 0
+    ? ['#DE1029', '#DE1029']
+    : ['#e2e2e2', '#e2e2e2'];
```

Comment:
should we have default value as const for the progress color

Reply:
I added default value because if user delete progress_color key in config json, we still have default color.